### PR TITLE
Add RecoverAccountComfirm page

### PR DESCRIPTION
**Pull request by roshni73**
## Changes
- Add the RecoverAccoutConfirm page 

## This PR doesn't introduce:
- [x] typos
- [x] conflict markers
- [x] unwanted comments
- [x] temporary files, auto-generated files or secret keys
- [x] `console.log` meant for debugging
- [x] codegen errors

### CODE REVIEW

**Review by barshathakuri — APPROVED** [review]

File: src/App/redirects/RecoverAccountRedirects.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        : routes.pageNotFound.path;
+
+    if (resetPasswordLink) {
+        navigate(resetPasswordLink);
+    }
+
+    return null;
+}
+
+Component.displayName = 'RecoverAccountRedirects';
```
Comment (AdityaKhatri) [inline src/App/redirects/RecoverAccountRedirects.tsx]:
```suggestion
Component.displayName = 'RecoverAccountRedirect';
```

File: src/App/redirects/RecoverAccountRedirects.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    useNavigate,
+    useParams,
+} from 'react-router-dom';
+
+import routes from '#routes';
+
+interface ResetPasswordParams {
+    userId: string | undefined;
+    resetToken: string | undefined;
+    [key: string]: string | undefined;
```
Comment (AdityaKhatri) [inline src/App/redirects/RecoverAccountRedirects.tsx]: not necessary

File: src/views/RecoverAccountConfirm/index.tsx [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+
+        return baseSchema;
+    },
+};
+
+const defaultFormValue: FormFields = {};
+
+// eslint-disable-next-line import/prefer-default-export
+export function Component() {
+    const { userId, resetToken } = useParams<{ userId: string, resetToken: string }>();
```
Comment (AdityaKhatri) [inline src/views/RecoverAccountConfirm/index.tsx]:
```suggestion
    const { userId, resetToken } = useParams<{ userId?: string, resetToken?: string }>();
```

File: src/views/RecoverAccountConfirm/index.tsx [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+        (formValues: FormFields) => {
+            passwordResetConfirm({
+                variables: {
+                    data: {
+                        newPassword: formValues.newPassword,
+                        captcha: formValues.captcha,
+                        token: resetToken,
+                        uuid: userId,
+                    } as UserPasswordResetConfirmInput,
+                },
```
Comment (AdityaKhatri) [inline src/views/RecoverAccountConfirm/index.tsx:177]: check if token and uuid is there or not

**Review by AdityaKhatri — CHANGES_REQUESTED** [review]

**Pull request merged**